src/Hill.jl

```diff
@@ -45,20 +45,37 @@ end
 import LineSearches
 
 
-function optimize_helper(f, g!, low::Vector, high::Vector, maxstep::Int)
-    initial_x = low + (high - low) / 3.0
-
+function optimize_helper(f, g!, low::Vector, high::Vector, x0::Vector, maxstep::Int)
     method = Fminbox(LBFGS(linesearch = LineSearches.BackTracking()))
 
     options = Optim.Options(outer_iterations = 2, show_trace = true, iterations = maxstep)
-    results = optimize(f, g!, low, high, initial_x, method, options)
+    results = optimize(f, g!, low, high, x0, method, options)
 
     println(results)
 
     return Optim.minimum(results), Optim.minimizer(results)
 end
 
 
+function optimize_helper(f, g!, low::Vector, high::Vector, maxstep::Int)
+    sobolPoints = 10
+    s = SobolSeq(low, high)
+    skip(s, sobolPoints)
+
```

Comment:
Thank you so much. I will continue based on this.
One small question. These two functions have the same name. I think the second is calling the first one. I can change the name if it is okay.

Reply:
Yep. Which one gets called depends on whether you provide an `x0` argument.

Reply:
Changing the name is fine.